packages/grid/x-grid/src/tests/rows.DataGridPro.test.tsx

```diff
@@ -384,51 +384,51 @@ describe('<DataGridPro /> - Rows', () => {
       );
     };
 
-    it('Rows should not be virtualized when the number of rows fit in the viewport', () => {
-      const headerHeight = 50;
-      const rowHeight = 50;
-      const maxRowsNotVirtualized = (300 - headerHeight) / rowHeight;
-      render(
-        <TestCaseVirtualization
-          nbRows={maxRowsNotVirtualized}
-          hideFooter
-          headerHeight={headerHeight}
-          rowHeight={rowHeight}
-        />,
-      );
-
-      const isVirtualized = apiRef.current.state.containerSizes!.isVirtualized;
-      expect(isVirtualized).to.equal(false);
-    });
-
-    it('Rows should be virtualized when at least 2 rows are outside the viewport', () => {
-      const headerHeight = 50;
-      const rowHeight = 50;
-      const maxRowsNotVirtualized = (300 - headerHeight) / rowHeight;
-      render(
-        <TestCaseVirtualization
-          nbRows={maxRowsNotVirtualized + 1}
-          hideFooter
-          headerHeight={headerHeight}
-          rowHeight={rowHeight}
-        />,
-      );
-
-      let isVirtualized = apiRef.current.state.containerSizes!.isVirtualized;
-      expect(isVirtualized).to.equal(false);
-
-      render(
-        <TestCaseVirtualization
-          nbRows={maxRowsNotVirtualized + 2}
-          hideFooter
-          headerHeight={headerHeight}
-          rowHeight={rowHeight}
-        />,
-      );
-
-      isVirtualized = apiRef.current.state.containerSizes!.isVirtualized;
-      expect(isVirtualized).to.equal(true);
-    });
+    // it('Rows should not be virtualized when the number of rows fit in the viewport', () => {
```

Comment:
I guess it's like this because it's a WIP.

Reply:
I would like to see with @m4theushw on this one.
I don't know how to check if the virtualization is on or off.
Since the new virtualization, these test do not check the right think, they check a state that was triggering the old virtualization but that is not used anymore.

Reply:
Removed